denis/configure.py

```diff
@@ -97,16 +100,18 @@ def remove(assignment):
         print(f'no such assignment {assignment}')
 
 
-def dump():
-    def timestamp_to_isoformat(timestamp):
-        return datetime.fromtimestamp(timestamp).astimezone().isoformat()
+def dump(fmt_iso):
+    def timestamp_to_formatted(timestamp):
+        fmt = '%A, %B %d, %Y %I:%M:%S %p %Z (%z)'
+        dt = datetime.fromtimestamp(timestamp).astimezone()
+        return dt.isoformat() if fmt_iso else dt.strftime(fmt)
 
     print(' --- Assignments ---')
     for asn in db.Assignment.select():
         print(f'''{asn.name}:
-\tInitial:\t{timestamp_to_isoformat(asn.initial_due_date)}
-\tPeer Review:\t{timestamp_to_isoformat(asn.peer_review_due_date)}
-\tFinal:\t\t{timestamp_to_isoformat(asn.final_due_date)}''')
+\tInitial:\t{timestamp_to_formatted(asn.initial_due_date)}
+\tPeer Review:\t{timestamp_to_formatted(asn.peer_review_due_date)}
+\tFinal:\t\t{timestamp_to_formatted(asn.final_due_date)}''')
```

Comment:
kind of a nitpick, but I would prefer if the function didn't need to be renamed as part of this commit, could just call it this from the start and it would make this diff clearer